crates/llama-cpp-server/src/lib.rs

```diff
@@ -89,6 +89,10 @@ impl CompletionServer {
             context_size,
         );
         server.start().await;
+        Self::new_with_server(Arc::new(server)).await
+    }
+
+    async fn new_with_server(server: Arc<LlamaCppSupervisor>) -> Self {
```

Comment:
```suggestion
    async fn new_with_supervisor(server: Arc<LlamaCppSupervisor>) -> Self {
```

nit: to be consistent with struct naming